src/stylesheets/core/mixins/_icon.scss

```diff
@@ -116,15 +116,16 @@
     map-get($icon-object, "path"),
     $theme-image-path
   );
-  $ie11-variant: get-color-token-from-bg($contrast-bg, $color-variant, "black");
-  $filename-ie11-variant: if($ie11-variant == "black", null, $ie11-variant);
-  $filename: if(
-    $filename-ie11-variant,
-    "#{$filename-base}-#{$filename-ie11-variant}.svg",
-    "#{$filename-base}.svg"
+  $ie11-variant: get-color-token-from-bg($contrast-bg, "white", "black");
+  $filename-ie11: if(
+    $ie11-variant == "white",
+    "usa-icons-bg/#{$filename-base}--white.svg",
+    "usa-icons/#{$filename-base}.svg"
   );
 
-  $image-props: url("#{$path}/#{$filename}") no-repeat center / #{$width} #{$height};
+  $mask-props: url("#{$path}/#{$filename-base}.svg") no-repeat center / #{$width}
+    #{$height};
+  $image-props: url("#{$path}/#{$filename-ie11}") no-repeat center / #{$width} #{$height};
```

Comment:
Fix this code so it pulls the right `--white` variant from `usa-icons-bg` for IE support